static/js/filter.js

```diff
@@ -226,13 +226,25 @@ Filter.parse = function (str) {
             // FIXME: Should we skip unknown operator names here?
```

Comment:
I think probably the `FIXME` comment can be removed now, since we've fixed this, right?